osgi/jta/pom.xml

```diff
@@ -206,7 +212,7 @@
 						   osgi.service;effective:=active;objectClass=javax.transaction.UserTransaction
 					   </Provide-Capability>
 					   <Require-Capability>
-						   osgi.service;effective:=active;resolution:=optional;cardinality:=multiple;filter:=(objectClass=org.jboss.tm.XAResourceRecovery)
+						   osgi.service;effective:=active;resolution:=optional;filter:=(objectClass=org.jboss.tm.XAResourceRecovery)
```

Comment:
Do you recall why we had cardinality before? Is it from a different version of OSGi spec maybe?